### ISSUE TITLE
Fix typo in comment in validation.properties files

### ISSUE THREAD

**Issue by kwwall** [ESAPI/esapi-java-legacy#808]
There is a typo where 2 consecutive lines have 'and' at the end of the one line, followed by an 'and' at the start of the next line, thereby having a double 'and'. Specifically, the lines that should be fixed by removing the " and " at the end of these lines are:

```bash
$ grep -n 'and $' $(find . -name validation.properties)
./configuration/esapi/validation.properties:17:# canonicalization. It is preferable to use the "get" methods (which throw exceptions) **and** 
./src/test/resources/esapi/validation.properties:30:# canonicalization. It is preferable to use the "get" methods (which throw exceptions) **and** 
```

**Comment by kwwall** [comment]
@robstoll is working on this as part of PR #807

**Comment by kwwall** [comment]
Closed via PR #807.
